index.bs

```diff
@@ -68,17 +69,81 @@ Examples {#examples}
+    it is printed to the console.
+
     <pre highlight="js">
-    let sensor = new AmbientLightSensor();
+    const sensor = new AmbientLightSensor();
+    sensor.onreading = () => console.log(sensor.illuminance);
+    sensor.onerror = event => console.log(event.error.name, event.error.message);
     sensor.start();
-    
-    sensor.onreading = function(event) {
-        console.log(sensor.illuminance);
-    };
+    </pre>
+</div>
 
-    sensor.onerror = function(event) {
-        console.log(event.error.name, event.error.message);
+<div class="example">
+    In this example, exposure value (EV) at ISO 100 is calculated from
+    the ambient light [=sensor readings=]. Initially, we check that the user
+    agent has permissions to access ambient light [=sensor readings=]. Then,
+    {{AmbientLightSensor/illuminance!!attribute}} value is converted to the
+    closest exposure value.
+
+    <pre highlight="js">
+    navigator.permissions.query({ name: 'ambient-light-sensor' }).then(result => {
+        if (result.state === 'denied') {
+            console.log('Permission to use ambient light sensor is denied.');
+            return;
+        }
+
+        const als = new AmbientLightSensor({frequency: 20});
+        als.addEventListener('activate', () => console.log('Ready to measure EV.'));
+        als.addEventListener('error', error => console.log(\`Error: ${error.name}\`));
+        als.addEventListener('reading', () => {
+            // Defaut ISO value.
+            const ISO = 100;
+            // Incident-light calibration constant.
+            const C = 250;
+
+            let EV = Math.round(Math.log2((als.illuminance * ISO) / C));
+            console.log(\`Exposure Value (EV) is: ${EV}\`);
+        });
+
+        als.start();
+    });
+    </pre>
+</div>
+
+<div class="example">
+    This example demonstrates how ambient light [=sensor readings=] can be mapped
+    to recommended workplace light levels.
+
+    <pre highlight="js">
+    const als = new AmbientLightSensor();
+
+    als.onreading = () => {
+        let str = luxToWorkplaceLevel(als.illuminance);
+        if (typeof str !== 'undefined') {
+            console.log(\`Light level is suitable for: ${str}.\`);
+        }
     };
+
+    als.start();
+
+    function luxToWorkplaceLevel(lux) {
+        if (lux > 20 && lux < 100) {
+            return 'public areas, short visits';
+        } else if (lux > 100 && lux < 150) {
+            return 'occasionally performed visual tasks';
+        } else if (lux > 150 && lux < 250) {
+            return 'easy office work, classes, homes, theaters';
+        } else if (lux > 250 && lux < 500) {
+            return 'normal office work, groceries, laboratories';
+        } else if (lux > 500 && lux < 1000) {
+            return 'mechanical workshops, drawing, supermarkets';
+        } else if (lux > 1000 && lux < 5000) {
+            return 'detailed drawing work, visual tasks of low contrast';
+        }
```

Comment:
Perhaps good to explicitly return here (with and without will return `undefined`):

`return;`

Reply:
done